samwise.js

```diff
@@ -8,54 +8,59 @@
 
```

Comment:
SCRIPT-Consider using more descriptive naming for this boolean variable to clarify its purpose.